Project-1/README.md

```diff
@@ -255,3 +255,5 @@ Proof: (¬P ∨ ¬Q) ≡ ¬(P ∧ Q)
 10. Compare and contrast the proofs provided for (a) in your answers to questions 8 and 9. Explain the different assumptions, strategies, etc. exhibited in tree proofs vs natural deduction proofs. 
 
 The tree proof is more neater and easier to follow than the natural deduction proof. 
+
```

Comment:
Question 2. A tautology's CNF is any formula of the form of "p∨¬p", so you can write "B∨not B" or "A∨not A" or something else.

Q3: Today our discussion results are as follows:

The following are results of our today's meeting:

(a) B(x,y)= ∃u(P(u,x)∧P(u,y)∧¬(x=u)∧¬(y=u))∧¬F(x)∧¬(x=y).

A guy cannot be a brother or parent of himself/herself.

(b) A(x,y)= ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)

(c) C(x,y)= ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x=y)

(d) O(x)= ∃u(P(u,x)∧∀y(P(u,y)→x=y))

(e) T(x)= ∃u∃y∃z(¬(y=z)∧¬(x=y)∧¬(x=z)∧P(u,x)∧P(u,y)∧P(u,z)∧¬F(y)∧¬F(z))∧∀w(P(u,w)∧¬F(w)→ w=x ∨ w=y ∨ w=z)

Q4. I will give an example. (a) $B ≡ M\sqcap ∃parentOf^-.(∃parentOf\ge2)$. This is my definition. It means: x is a brother of y = x is a male & x is a child of some guy, who is a parent of at least 2 children. The key point is to introduce $parentOf^-$, which is the reverse of the parenthood, namely childhood.

Q5. "T ⊑ ¬ O 
	B ⊑ ¬ O"

These two formulas are not from Q4 because Q4's formulas are all equivalence formulas (a definition is an equivalence formula). 

Also, you don't tell us What is K. In order to do this, you should assign a set of objects to each concept, like C={a, b, c} 

Q6. (b) "the former means that for all y there exist a x and a z, and the latter means that for all x and y there exists a y. "

This is wrong. ∃x∀y∃z means “there exists x such that for all y, there exists z such that” while ∀x∃y∀z means "for all x, there exists y such that for all z". Notice: the order of variables x, y, z is very important. You cannot violate their order.

(c) has the same problem as (b). Please revise it.

Q7. In order to represent a graph, you can use Mermaid, or use words to describe the nodes and edges of that graph. For example, you can say like this:

Verticles: a, b, c

Edges: ab, ac, aa.

9. When you say "|6. ⊥                ⊥I 1,4,5", this step implicitly uses DeMorgan's laws.

Your "second step" is actually a correct proof of 9(a), congrats! But you don't give proofs of 9(b) and 9(d).

10. Notice: Tree proof uses only one rule: indirect derivation, but natural deduction can use multiple rules